src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Route, Routes } from 'react-router-dom';
+import Detailspage from './components/Detailspage/Detailspage';
+import Homepage from './components/Homepage/Homepage';
+import Navbar from './components/Navbar/Navbar';
+import { getConis } from './redux/Homepage/homepage';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    console.log('in useEffect');
```

Comment:
- [ ] Kindly ensure to remove `console.log` from your code, as mentioned in the previous review.